### PR TITLE
DESK-370 Clearer airbrake messages

### PR DESCRIPTION
**Pull request by JamieRuderman**
So we can attempt to filter out the heartbeat errors.

### CODE REVIEW

File: backend/src/Command.ts [hunk 1]
```diff
@@ -59,7 +62,10 @@ export default class Command {
         result = stdout.toString()
       }
     } catch (error) {
-      AirBrake.notify({ message: 'COMMAND ERROR', command: this.toString(), error })
+      AirBrake.notify({
+        params: { type: 'COMMAND ERROR', command: this.toString() },
+        error: error,
```
Comment (danawoodman) [inline backend/src/Command.ts]:
```suggestion
        error,
```

File: backend/src/index.ts [hunk 2]
```diff
@@ -26,16 +26,22 @@ Logger.info('Desktop starting up!')
...
 process
   .on('uncaughtException', (error: Error) => {
-    d('Caught exception', error)
-    AirBrake.notify({ message: 'CAUGHT EXCEPTION', error })
-    Logger.warn('CAUGHT EXCEPTION', { error, details: error.toString() })
-    EventBus.emit(CLI.EVENTS.error, error.toString())
+    d('Uncaught exception', error)
+    AirBrake.notify({
+      params: { type: 'UNCAUGHT EXCEPTION' },
+      error: error,
```
Comment (danawoodman) [inline backend/src/index.ts]:
```suggestion
      error,
```

Comment (JamieRuderman) [reply backend/src/index.ts]: fixed

File: backend/src/Command.ts (hunk 1 above)
Comment (JamieRuderman) [reply backend/src/Command.ts]: yes. fixed.

**Pull request merged**